build.gradle

```diff
@@ -74,18 +90,32 @@ task sourcesJar(type: Jar) {
 task groovydocJar(type: Jar) {
     classifier = 'groovydoc'
     from groovydoc
-} 
+}
 
 artifacts {
     archives groovydocJar, sourcesJar
 }
 
 if (!version.endsWith("SNAPSHOT")) {
-    signing {           
+    signing {
         sign configurations.archives
     }
 }
 
+pluginBundle {
+  website = 'https://github.com/WASdev/ci.gradle'
+  vcsUrl = 'https://github.com/WASdev/ci.gradle'
+  description = 'Gradle plugins for managing Liberty profile servers'
```

Comment:
Remove "profile"

Reply:
also "plugin" rather than "plugins"

Reply:
Took this from the description on the github page. Went ahead and changed this as well.